### PR TITLE
Split `_cap_el`

### PR DESCRIPTION
**Pull request by eugulixes**

### CODE REVIEW

File: gits/terminal.py [hunk 1]
```diff
@@ -366,12 +366,14 @@ def _cap_ed(self, l=None):
         self._zero((self._cur_x, self._cur_y), (self._cols, self._rows - 1))
 
     def _cap_el(self, l=None):
-        """Clears a line from the cursor position to the end of the line. """
+        """Clears from the current cursor position to the end of the line
```
Comment (dshil) [inline gits/terminal.py]: s/Clears/Clears the screen/
It seems that we miss the *screen* word.

File: gits/test/terminal_test.py [hunk 2]
```diff
@@ -916,8 +916,8 @@ def _check_cap_el(self, pos, s):
         term._cap_rs1()
 
     def test_cap_el(self):
-        """The emulator should have the possibility to clear the screen from
-        the current cursor position to the end of line.
+        """The emulator should have the possibility to clear from the current
```
Comment (dshil) [inline gits/test/terminal_test.py]: s/clear from/clear the screen from/
It seems that we miss the *screen* word.

File: gits/test/terminal_test.py [hunk 3]
```diff
@@ -956,8 +956,8 @@ def _check_cap_el1(self, pos, s):
         term._cap_rs1()
         
     def test_cap_el1(self):
-        """The emulator should have the possibility to clear the screen from the
-        beginning of the line to current cursor position.
+        """The emulator should have the possibility to clear from the beginning
```
Comment (dshil) [inline gits/test/terminal_test.py]: Why do you omit the *screen* word?

Comment (eugulixes) [reply gits/test/terminal_test.py]: IMO it's redundant here.

File: gits/test/terminal_test.py (hunk 2 above)
Comment (eugulixes) [reply gits/test/terminal_test.py]: s/clear from/clear the line from/

Comment (dshil) [reply gits/test/terminal_test.py]: Let's use your suggestion.

File: gits/test/terminal_test.py (hunk 3 above)
Comment (dshil) [reply gits/test/terminal_test.py]: Yep, but omitting the *line* word is not redundant. Agree?

**Comment by dshil** [comment]
LGTM

**Review by dshil — APPROVED** [review]
LGTM

**Pull request merged**